data/json/mapgen/musicstore.json

```diff
@@ -1,4 +1,41 @@
 [
+  {
+    "id": "musicstore_showpiece",
+    "type": "item_group",
+    "items": [
+      {
+        "distribution": [
+          { "item": "acoustic_guitar", "prob": 100 },
+          { "item": "bagpipes", "prob": 25 },
+          { "item": "banjo", "prob": 60 },
+          { "item": "bone_flute", "prob": 20 },
+          { "item": "clarinet", "prob": 50 },
+          { "item": "flute", "prob": 50 },
+          { "item": "harmonica_holder", "prob": 70 },
+          { "item": "jug_clay", "prob": 20 },
+          { "item": "mp3", "prob": 20 },
+          { "item": "saxophone", "prob": 50 },
+          { "item": "trumpet", "prob": 50 },
+          { "item": "tuba", "prob": 25 },
+          { "item": "ukulele", "prob": 50 },
+          { "item": "violin", "prob": 90 },
+          { "item": "violin_golden", "prob": 15 },
+          {
+            "collection": [
+              { "item": "spoon" },
```

Comment:
Spoons as musical instrument are made of wood, as far as I recall, and spoons in-game are made of steel.

Reply:
Pun intended?